pkgs/development/python-modules/xlib/default.nix

```diff
@@ -22,6 +22,10 @@ buildPythonPackage rec {
     sha256 = "155p9xhsk01z9vdml74h07svlqy6gljnx9c6qbydcr14lwghwn06";
   };
 
+  patches = [
+    ./fix-no-protocol-specified.patch
```

Comment:
Is there an upstream patch we could fetch?

Reply:
I don't think so. Should I remove this until there is a patch upstream?

Reply:
No but we should try to upstream it.

Reply:
Submitted upstream in https://github.com/python-xlib/python-xlib/pull/230.